### ISSUE TITLE
Allow multiple <routes> on the same page

### ISSUE THREAD

**Issue by ghempton** [ghempton/ember-react#2]
This is a potential scenario when dealing with multiple `<routes>` components mounted within Ember's router.

This will be fixed once https://github.com/rackt/react-router/pull/414 is merged.

**Comment by ghempton** [comment]
No longer relevant with latest ReactRouter
